build.gradle

```diff
@@ -34,7 +34,7 @@ subprojects {
     apply plugin: 'findbugs'
 
     group = 'io.minio'
-    version = '3.0.4'
+    version = '3.0.5'
```

Comment:
Yes it is basically since we made a release, we move the current master to the next.